gradebookng/tool/src/webapp/styles/gradebook-importexport.css

```diff
@@ -36,4 +36,8 @@
 #gradebookImportExport .gb-import-export-section p,
 #gradebookImportExport .gb-import-export-section form {
   margin: 1em 0 0;
+}
+
+.no_changes {
+	color: grey;
```

Comment:
feel free to change this. i went with a boring grey. See screenie:
![screen shot 2016-05-31 at 11 48 10 pm](https://cloud.githubusercontent.com/assets/129129/15676597/3088ed6a-278a-11e6-869f-c92804e4c6fc.png)